metrics-operator/config/crd/bases/metrics.keptn.sh_keptnmetrics.yaml

```diff
@@ -1,278 +0,0 @@
----
```

Comment:
why did you remove his file?

Reply:
@RealAnna requested I remove the file

Reply:
no she didn't, she just requested not to commit the changes you made tot hat file, but not to delete the whole file

Reply:
Oh Alright I will recreate the file.

Reply:
I've added the file.